sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/ConfigurationClientBuilder.java

```diff
@@ -239,25 +239,29 @@ public ConfigurationClientBuilder endpoint(String endpoint) {
      * @param connectionString Connection string in the format "endpoint={endpoint_value};id={id_value};
      * secret={secret_value}"
      * @return The updated ConfigurationClientBuilder object.
-     * @throws NullPointerException If {@code credential} is {@code null}.
+     * @throws NullPointerException If {@code connectionString} is {@code null}.
+     * @throws IllegalArgumentException if {@code connectionString} is an empty string. Or the secret is invalid and
```

Comment:
_No change required in this PR._

Do our Checkstyle rules need to be updated to catch this missing case? It looks like the `IllegalArgumentException` has been getting thrown for a while but never documented, we have a Checkstyle rule that should failed PR validation due to this.

Reply:
created an issue to investigate it: https://github.com/Azure/azure-sdk-for-java/issues/7147